industry_benchmarks/utils/data_gathering.py

```diff
@@ -374,31 +571,41 @@ def gather_ligand_scores(
     help="Path to the output directory that stores all data.",
 )
 @click.option(
-    '--fixed_ligand_network',
+    '--fixed_alchemical_network',
     type=click.Path(dir_okay=False, file_okay=True, path_type=pathlib.Path),
-    default=pathlib.Path("./ligand_network.graphml"),
+    default=None,
     required=False,
     help=("Only needed when a broken network was fixed with additional edges. "
-          "Path to the ligand_network.graphml file that was used to run the "
+          "Path to the alchemical_network.json file that was used to run the "
          "simulations of fixing the network."),
 )
+@click.option(
+    "--results-folder",
+    type=click.Path(dir_okay=True, file_okay=False, path_type=pathlib.Path),
+    multipule=True,
+    help="The path to the directory which contains transformation results, can be supplied multipule times for each repeat folder."
+)
 def gather_data(
-    input_ligand_network,
-    output_dir,
-    fixed_ligand_network,
+    input_alchemical_network: pathlib.Path,
+    output_dir: pathlib.Path,
+    fixed_alchemical_network: None | pathlib.Path,
+    results_folder: list[pathlib.Path],
```

Comment:
@hannahbaumann Is changing this to take the alchemical network ok? I needed a way to ensure I could get the transform names if users had used the old version of the cleanup script which removed the input components and followed the plan in the fix networks script.

Reply:
Yes, that should be ok! Thanks, I didn't think about this!